node/core/backing/src/lib.rs

```diff
@@ -1560,6 +1632,44 @@ async fn import_statement<Context>(
 					"Candidate backed",
 				);
 
+				let head_data_hash = backed.candidate.descriptor().para_head;
+				if let Entry::Occupied(entry) =
+					seconding_check_requests.entry((para_id, head_data_hash))
+				{
+					if maybe_membership.is_none() {
+						let (tx, rx) = oneshot::channel();
+						ctx.send_message(ProspectiveParachainsMessage::GetTreeMembership(
+							para_id,
+							candidate_hash,
+							tx,
+						))
+						.await;
+
+						match rx.await {
+							Err(oneshot::Canceled) => gum::warn!(
+								target: LOG_TARGET,
+								?para_id,
+								?candidate_hash,
+								"Prospective parachains subsystem unreachable for membership request",
```

Comment:
Not sure this is the right log message. Can this happen if the relay parent goes out of view ?

Reply:
> Not sure this is the right log message

Why? If the candidate relay parent went out of view of prospective parachains (and not out of ours due to race), it leaves all fragment trees and we would receive empty membership. Whatever happens next isn't important, all subsystems will soon remove this candidate.